mne/preprocessing/ica.py

```diff
@@ -1339,7 +1339,7 @@ def _pick_sources(self, data, include, exclude):
         if self.noise_cov is None:  # revert standardization
             data *= self._pre_whitener
         else:
-            data = fast_dot(linalg.pinv(self._pre_whitener), data)
+            data = fast_dot(linalg.pinv(self._pre_whitener, cond=1e-14), data)
```

Comment:
LGTM but usually `linalg.lstsq` is a better solution (numerically and possibly speed-wise) compared to `np.dot(pinv(...), ...)`, you can investigate here or we can merge this as-is